docs/src/components/pricing/FAQ.tsx

```diff
@@ -92,14 +90,13 @@ const faqData = [
     summary: 'Am I allowed to use the product after the update entitlement expires?',
     detail: (
       <React.Fragment>
-        Yes. The license is perpetual, so you are allowed to continue using the product even after
-        the entitlement expires. However, you will be using the last product version released before
-        this time. You will lose access to subsequently released functionality and updates, as well
-        as technical support.
+        Yes. You're allowed to continue using any version of the product in production environments,
+        even after the entitlement expires. However, you will be required to renew your license if
+        you wish to continue development, and to access subsequently released functionality and
+        updates, as well as technical support.
```

Comment:
Simplifying the language a bit.

```suggestion
        Yes. 
        You can continue to use the product in production environments after the entitlement expires.
        But you will need to renew your license to continue development, update for new features, or gain access to technical support.
```